src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -102,30 +102,31 @@ export function libraryCurrent() {
 }
 
 export function libraryPagerGen() {
-  numCards = libraryNumActiveCards();
-  pages = Math.ceil(numCards / itemsPerPage);
+  const numCardsCount = libraryNumActiveCards();
+  numCards = libraryNumCards();
+  pages = Math.ceil(numCardsCount / itemsPerPage);
   if (document.getElementById('pager-nums-insert')) {
     const diff = pages - activePage;
     let pagerHtml;
     // This is our active page / pager button.
     pagerHtml = `<a href="#${activePage}" aria-label="Page ${activePage}" aria-current="true" id="va-pagination-active-num" class="va-pagination-active a-page-numbers pager-focus-item">
     ${activePage}</a>`;
-    // If we have more than one page, add a button in front of active button.
-    if (diff > 1 && (numCards.first === undefined || numCards.first > 9)) {
+    // If we have more than one page, add a button in front of active button
+    if (diff >= 1 && (numCards.first === undefined || numCards.last > 9)) {
```

Comment:
Why are we checking `numCards.last > 9`? Does the number of cards on a page matter if the number of pages has already been calculated?

Reply:
Additionally, why keep the check for `numCards.first === undefined`? It seems pretty clear that this was initially added to accommodate the attempt to check an integer for a property (so it's always undefined). Now that `numCards` is correctly initialized to an object with properties `first` and `last`, would it ever be undefined? And, even if it would be, why would that affect the rendering of the pager?

Reply:
removed the extra conditionals

Reply:
@ryguyk thoughts on next steps (approval vs. new feedback)?  Keeping my eye on this PR as it's a sprint goal.